test/integration/targets/git/tasks/main.yml

```diff
@@ -27,7 +27,11 @@
 - include: change-repo-url.yml
 - include: depth.yml
 - include: checkout-new-tag.yml
-- include: tag-verification.yml
+- include: gpg-verification.yml
+  when: >
+    not gpg_version.stderr and
+    gpg_version.stdout and
```

Comment:
That old check were kind of weird.

https://github.com/ansible/ansible/blob/v2.3.1.0-1/test/integration/targets/git/tasks/tag-verification.yml#L10-L11

The `or` logic resulting in a new enough GnuPG version negating the need for a new enough Git version.